### PR TITLE
Fix Duo API change - follow result_url and return cookie from result

### PR DESCRIPTION
**Pull request by bghinkle**
Fixes https://github.com/venth/aws-adfs/issues/105 (hat tip to https://github.com/segmentio/aws-okta/pull/68/ for hints about the logic)

### CODE REVIEW

**Comment by bghinkle** [comment]
It looks like Python 3.3 builds are failing because setuptools now requires a minimum of Python 3.4. Not related to the changes in this PR.

**Comment by KyleJamesWalker** [comment]
Thanks for this!!!! Eagerly awaiting merge! 💃

File: aws_adfs/_duo_authenticator.py [hunk 1]
```diff
@@ -173,10 +173,54 @@ def _authentication_result(
...
+
+    if response.status_code != 200:
+        raise click.ClickException(
+            u'Issues when following the Duo result URL after '
+            u'authentication. The error response {}'.format(
+                response
+            )
+        )
+    json_response = response.json()
+    print
```
Comment (KyleJamesWalker) [inline aws_adfs/_duo_authenticator.py]: Did you mean to leave this print line in here?

**Comment by KyleJamesWalker** [comment]
@bghinkle Can you remove this line https://github.com/venth/aws-adfs/blob/master/.travis.yml#L4 for @venth to easily approve and merge this PR?

File: aws_adfs/_duo_authenticator.py (hunk 1 above)
Comment (bghinkle) [reply aws_adfs/_duo_authenticator.py]: Whoops, forgot to save that before committing. It's gone in newest commit.

**Comment by bghinkle** [comment]
I've removed the Python 3.3 support in travis.yml and now the build is passing.

**Comment by venth** [comment]
Merged and released with version 0.12.1. Thank you guys 👍

**Pull request merged**